examples/Console/TestOtlpExporter.cs

```diff
@@ -28,6 +28,24 @@ internal static object Run(string endpoint)
 
         private static object RunWithActivitySource(string endpoint)
         {
+            /*
+             * Setup an OpenTelemetry Collector to run on local docker.
+             *
+             * Open a terminal window and run the OpenTelemetry Collector with an OTLP receiver, by running:
+             *
+             *     docker run --rm -it -p 55680:55680 -v $(pwd):/cfg otel/opentelemetry-collector:0.7.0 --config=/cfg/opentelemetry-collector-otlp-example.yaml
```

Comment:
C:\Users\cithomas>docker run --rm -it -p 55680:55680 -v $(pwd):/cfg otel/opentelemetry-collector:0.7.0 --config=/cfg/opentelemetry-collector-otlp-example.yaml

Unable to find image 'otel/opentelemetry-collector:0.7.0' locally
0.7.0: Pulling from otel/opentelemetry-collector
551e704fb86a: Pull complete                                                                                             3fe0fe77c256: Pull complete                                                                                             7adce1a556bc: Pull complete                                                                                             Digest: sha256:8182de36e15c2121c7215b5b1678c37e16badaa70094ab5fcc1015b83632976f
Status: Downloaded newer image for otel/opentelemetry-collector:0.7.0

docker: Error response from daemon: create $(pwd): "$(pwd)" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.

I get the above error when I run this in windows machine.

Reply:
Let me get hold of a Windows box to try this...

Reply:
🤦 of course `$(pwd)` is not going to work on Windows ...

Reply:
@cijothomas could you try replacing `$(pwd)` with `%cd%`? Hopefully docker will replace any invalid char...

Reply:
Yes! That solves the command issue. 

docker run --rm -it -p 55680:55680 -v %cd%:/cfg otel/opentelemetry-collector:0.7.0 --config=/cfg/opentelemetry-collector-otlp-example.yaml

Where would I get this yaml file? Sorry for being lazy here, but I am validating how 'self-contained' the examples are 😄

Reply:
I'm adding it in the PR :) see below. Let me update the instructions with Windows instructions.

Reply:
@cijothomas could you please double-check the updated command for Windows? I added double-quotes in case the path has spaces, if you could test in such a path it will be great.

Reply:
Yes this works.